### ISSUE TITLE
java.lang.IllegalStateException: Cannot access request header fields after connection is set

### ISSUE THREAD

**Issue by AndwareSsj** [lingochamp/FileDownloader#458]
```
java.lang.IllegalStateException: Cannot access request header fields after connection is set
     at com.android.okhttp.internal.http.HttpURLConnectionImpl.getRequestProperties(HttpURLConnectionImpl.java:168)
     at com.liulishuo.filedownloader.connection.FileDownloadUrlConnection.getRequestHeaderFields(FileDownloadUrlConnection.java:80)
     at com.liulishuo.filedownloader.exception.FileDownloadHttpException.<init>(FileDownloadHttpException.java:36)
     at com.liulishuo.filedownloader.services.FileDownloadRunnable.loop(FileDownloadRunnable.java:370)
     at com.liulishuo.filedownloader.services.FileDownloadRunnable.run(FileDownloadRunnable.java:222)
     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
     at java.lang.Thread.run(Thread.java:841)
```
error in 1.4.0

**Comment by gmribas** [comment]
@Jacksgong in which version will this fix be released?

**Comment by Jacksgong** [comment]
Sorry, these days are Spring Festival holidays, I was in the home with family, so delayed the time, I will fix #462 issue soon, then the 1.4.1 version will be released to fix this issue, the deadline is in the February 4.
